apps/visualization/views.py

```diff
@@ -45,3 +62,237 @@ def get_queryset(self):
             ) for value in result
         ])
         return DataCountryLevelMostRecent.objects.filter(filters).distinct('subvariable')
+
+
+class ExportRawDataView(ListAPIView):
+
+    filterset_class = DataGranularPublicFilter
+    queryset = DataGranularPublic.objects.all()
+
+    def get(self, request, format=None):
+        """
+        Export raw data
+        """
+
+        validate_export_params_together(self.request.query_params)
+        filterset = DataGranularPublicFilter(
+            data=self.request.query_params,
+            queryset=self.get_queryset(),
+        )
+        response = HttpResponse(
+            content_type='text/csv',
+            headers={'Content-Disposition': 'attachment; filename="raw-data.csv"'},
+        )
+        writer = csv.writer(response)
+        writer.writerow([
+            'emergency',
+            'country_name',
+            'region',
+            'iso3',
+            'admin_level_1',
+            'indicator_id',
+            'subvariable',
+            'indicator_name',
+            'thematic',
+            'thematic_description',
+            'topic',
+            'topic_description',
+            'indicator_description',
+            'type',
+            'question',
+            'indicator_value',
+            'nominator',
+            'error_margin',
+            'denominator',
+            'indicator_month',
+            'category',
+            'gender',
+            'age_group',
+            'age_info',
+            'target_group',
+            'indicator_matching',
+            'representativeness',
+            'limitation',
+            'indicator_comment',
+            'source_id',
+            'organisation',
+            'title',
+            'details',
+            'authors',
+            'methodology',
+            'sample_size',
+            'target_pop',
+            'scale',
+            'quality_check',
+            'access_type',
+            'source_comment',
+            'publication_channel',
+            'link',
+            'source_date',
+            'sample_type',
+        ])
```

Comment:
`[field.name for field in ModelName_meta.get_fields()]` we can do this if we are importing all fields from the database.